depends/common/ogg/CMakeLists.txt

```diff
@@ -3,20 +3,20 @@ project(ogg)
 cmake_minimum_required(VERSION 2.8)
 include(CheckCCompilerFlag)
 
-CHECK_C_COMPILER_FLAG("-fPIC" HAVE_FPIC)
-if(HAVE_FPIC)
+if(NOT WIN32)
```

Comment:
You can probably drop this condition and integrate everything into the `externalproject_add()` call since WIN32 uses its own `CMakeLists.txt`. The `CONFIGURE_COMMAND` wouldn't work on WIN32 either anyway.